helm/aws-pod-identity-webhook/templates/Deployment.yaml

```diff
@@ -1,19 +1,22 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: {{ .Values.name }}
+  name: {{ .Values.name }}-app
```

Comment:
do we really need to change the name here? just asking because we might need to change the prometheus rules again

Reply:
Yeah, because we're changing the matchSelectors. :( 

Otherwise we'd need to manually intervene in all upgrades.

Reply:
I'll update prometheus-rules before releasing this change.

Reply:
Thanks ❤️ , it might be not much. I haven't looked at it but I barely remember there was something